kokoro/scripts/test/go_test.sh

```diff
@@ -193,11 +193,6 @@ if [[ "${TEST_SUITE_NAME}" == "os_config_test" ]]; then
 fi
 
 STDERR_STDOUT_FILE="${KOKORO_ARTIFACTS_DIR}/test_stderr_stdout.txt"
-function produce_xml() {
-  cat "${STDERR_STDOUT_FILE}" | "$(go env GOPATH)/bin/go-junit-report" > "${LOGS_DIR}/sponge_log.xml"
-}
-# Always run produce_xml on exit, whether the test passes or fails.
-trap produce_xml EXIT
```

Comment:
This part might still be relevant, right? If the test crashes/panics, shouldn't we still get an XML file?

Reply:
oops, this isn't meant to be in this PR...

Reply:
Great, will re-raise in #1641.